bundles/org.openhab.binding.tellstick/src/main/feature/feature.xml

```diff
@@ -4,9 +4,10 @@
 
     <feature name="openhab-binding-tellstick" description="Tellstick Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <feature>openhab-runtime-jna</feature>
```

Comment:
How can we find out, if tellstick also works with JNA 5.4.0?

Reply:
We could create an issue for that and ask if all those that commented in https://github.com/openhab/openhab2-addons/issues/3849 can help with testing if it still works after updating the JNA dependency.

Reply:
Maybe @jannegpriv can test if the Tellstick Binding also works JNA 5.4.0?